core/src/env/test_env.rs

```diff
@@ -180,6 +186,7 @@ impl Default for TestEnvData {
             input: Vec::new(),
             random_seed: Vec::new(),
             now: Vec::new(),
+            current_block: Vec::new(),
```

Comment:
Also need to clear it in https://github.com/paritytech/ink/pull/132/files#diff-1fe109421ba68db891c2e06ae27fd2e9R203